mgmtsystem_audit/views/res_users.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
```

Comment:
@eugen-don [copyright in the views is part of the OCA template](https://github.com/OCA/maintainer-tools/blob/4679ce184af4d2af3175d0b52f3178b0c231b83c/template/module/views/model_name_view.xml#L2-L3).